regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -90,8 +90,8 @@ This contains all of the styles specific to the TOC
 
   a:link,
   a:visited {
-    color: @black;
-    .border-bottom-light (@width: 1px);
+    color: @navigation_gray;
+    .border-bottom-light (@width: 0px);
```

Comment:
would it make sense to bring back these overrides in ATF rather than leave one or two stray rules in notice-and-comment that are necessary to make the pilot design work?